spec/models/miq_policy_spec.rb

```diff
@@ -100,6 +100,16 @@
       end
     end
 
+    describe '#miq_event_definitions' do
+      before do
+        policy.miq_policy_contents.push(FactoryGirl.create(:miq_policy_content))
```

Comment:
@isimluk 

you can use `let` and `let!`
```ruby
let(:miq_policy_content) { FactoryGirl.create(:miq_policy_content) }
let(:policy)             { policy.miq_policy_contents.push(miq_policy_content) }

```
and rename it as you need. 

what do you think ?

and `let(:policy)` on line 90 is not usable for your case ?

there is great summary for writing specs 
http://betterspecs.org/